mri/operators/fourier/non_cartesian.py

```diff
@@ -34,6 +34,13 @@
                   "the master release. Till then you cannot use NUFFT on GPU")
     pass
 
+try:
+    from gpuNUFFT import NUFFTOp
+except ImportError:
+    warnings.warn("gpuNUFFT python package has not been found. If needed use "
+                  "the master release.")
```

Comment:
what do you mean by that?

Reply:
Will update the comment and point to README